node.gyp

```diff
@@ -920,6 +920,143 @@
         'node_dtrace_provider',
       ],
 
+      'variables': {
```

Comment:
Why 2 levels?

Reply:
Is it because of condition evaluation order.
In that case it's also possible to `>` or "late variable expansion operator", or just put a comment.

Reply:
@refack after trying to flatten the variables, I need to use `>` for late variable expansion and I found it's easy to get confused. I'd prefer 2 level variable and I will put a comment there. how do you think?

Reply:
👍
Comment is totally reasonable, but IMHO it's needed since it's a non-trivial pattern.